rpcserver.go

```diff
@@ -344,6 +345,36 @@ func handleAddNode(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (in
 	return nil, nil
 }
 
+// handleNode handles node commands.
+func handleNode(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (interface{}, error) {
+	c := cmd.(*btcjson.NodeCmd)
+
+	addr := normalizeAddress(c.Addr, activeNetParams.DefaultPort)
+	var err error
+	switch c.SubCmd {
+	case "disconnect":
+		err = s.server.DisconnectNode(addr)
+	case "remove":
+		err = s.server.RemoveNode(addr)
+	case "connect":
+		switch *c.ConnectSubCmd {
+		case "perm", "non-perm":
+			err = s.server.ConnectNode(addr, *c.ConnectSubCmd == "perm")
+		default:
+			err = errors.New("invalid subcommand for node connect")
+		}
+	default:
+		err = errors.New("invalid subcommand for node")
+	}
+
+	if err != nil {
+		return nil, internalRPCError(err.Error(), "")
```

Comment:
This is not really accurate either since many cases, such as an address that doesn't exist, are not internal errors.  They are simply `ErrRPCInvalidParameter`.  However, the returned errors from each of the functions above needs to be examined to determine what the type of errors that can be returned are to react accordingly.